charactersheet/charactersheet/services/common/account/messaging/chat_service.js

```diff
@@ -140,9 +140,6 @@ function _ChatService(config) {
         try {
             // Messages in Group Chats are ID'd by the group JID.
             var from = Strophe.getBareJidFromJid($(msg).attr('from'));
-            if (Strophe.getResourceFromJid(from) == null || Strophe.getResourceFromJid(from) === '') {
```

Comment:
Is this branch old?
It's missing this
https://github.com/adventurerscodex/adventurerscodex/blob/v1.4.2/charactersheet/charactersheet/services/common/account/messaging/chat_service.js#L150